### ISSUE TITLE
[SF_ACC] TabContainer: A11Y Issues

### ISSUE THREAD

**Issue by IrvinDelgado** [SAP/ui5-webcomponents#6049]
**Describe the bug**
When the TabContainer contains nested Tabs then NVDA doesn't announce the label properly such as "Item 1 of X"

**To Reproduce**
1. Go to [webcomponent-react-story](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-tabcontainer--default-story)
2. Scroll down to TabContainer with nested Tabs section
3. Open NVDA
4. Open the Speech Viewer for convenience 
5. Click on a tab container with nested tabs
6. Press the <kbd>&#8595;</kbd> While on a tabcontainer tab element
7. Notice that the label is not announced correctly as well as incorrect highlighting 

**Expected behavior**

Expect to work just like in the [Web-components Page](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/)

**Screenshots**
On Webcomponents-react (Unexpected)
<img width="1040" alt="IncorrectTabLabel" src="https://user-images.githubusercontent.com/62973847/200963884-09885f98-e781-413c-8dd4-2f97555a4e38.png">

On Webcomponents (Expected)
<img width="986" alt="CorrectTabLabel" src="https://user-images.githubusercontent.com/62973847/200963905-84ee4979-a8b9-4b81-ada4-648819416ed5.png">

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.9.0
`@ui5/webcomponents-react` version: 1.4.1
Operating System:
Browser: Chrome

**Additional context**

**Comment by dobrinyonkov** [comment]
Hi @IrvinDelgado,

NVDA automatically switches between Browse and Focus modes, but the user can toggle them using Insert + Space Bar.

The announcemnt comes from the fact that NVDA switches to browse mode (the "beap" sound) when the popup shows , which means that arrow navigation only moves the virtual cursor so that users can read all of the content iside the dialog. If you again swtich to the focus mode (INSERT + SPACE) you can again use the arrow keys as expected and described by the ui5 framework.

In our playground we have mistakenly set role="applcation" to the body which forces some rules on screen reader. We will remove that from our samples also #6053.
However, although as I mentioned, you can turn off the "automatic" mode switching (role="applcation") or with the screen reader settings, you really shouldn't be trying to force one mode or the other. The screen reader user should have control over that.

Kind Regards,
Dobrin
